src/zope/testrunner/tb_format.py

```diff
@@ -22,6 +22,13 @@
 import zope.testrunner.feature
 
 
+try:
+    from traceback import _sentinel
+except ImportError:
+    # before 3.12
+    _sentinel = False
```

Comment:
I would do 
```suggestion
    _sentinel = object()
```
so it can't be accidentally confused with passing a value of `False`, but I guess it probably doesn't matter.